sksparse/cholmod.pyx

```diff
@@ -145,6 +147,37 @@ cdef class Factor
 class CholmodError(Exception):
     pass
 
+class CholmodNotPositiveDefiniteError(CholmodError):
+    def __init__(self, message, column=None, factor=None):
+        super().__init__(message)
+        self.column = column
+        self.factor = factor
+
+class CholmodNotInstalledError(CholmodError):
+    def __init__(self, message=""):
+        message = "method not installed: {}".format(message)
```

Comment:
I had never a "not installed" error and there is no example in the documentation. It is just mentioned that it can be thrown.
I capitalized the first letter in each all error messages.